### PR TITLE
Inline documentation for updating campaign patches

### PR DESCRIPTION
**Pull request by eseliger**
image

### CODE REVIEW

File: web/src/enterprise/campaigns/detail/CampaignDetails.tsx [hunk 1]
```diff
@@ -425,6 +425,51 @@ export const CampaignDetails: React.FunctionComponent<Props> = ({
...
+                                        Using the{' '}
+                                        <a
+                                            href="https://github.com/sourcegraph/src-cli"
+                                            rel="noopener noreferrer"
+                                            target="_blank"
+                                        >
+                                            src CLI
+                                        </a>
+                                        , you can also apply a new patch set to an existing campaign. Following the
+                                        creation of the patch set with the
```
Comment (mrnugget) [inline web/src/enterprise/campaigns/detail/CampaignDetails.tsx:443]:
```suggestion
                                        creation of a new patch set that contains new patches, with the
```

File: web/src/enterprise/campaigns/detail/CampaignDetails.tsx [hunk 2]
```diff
@@ -425,6 +425,51 @@ export const CampaignDetails: React.FunctionComponent<Props> = ({
...
+                                                ),
+                                            }}
+                                        />
+                                    </div>
+                                    <p>
+                                        command, a URL will be output that will guide you to the web UI to allow you to
+                                        change an existing campaign’s patch set.
+                                    </p>
+                                    <p className="mb-0">
+                                        Also, check out the{' '}
```
Comment (mrnugget) [inline web/src/enterprise/campaigns/detail/CampaignDetails.tsx:460]:
```suggestion
                                        Take a look at the{' '}
```

File: web/src/enterprise/campaigns/detail/CampaignDetails.tsx [hunk 3]
```diff
@@ -425,6 +425,51 @@ export const CampaignDetails: React.FunctionComponent<Props> = ({
...
+                                        <a
+                                            href="https://docs.sourcegraph.com/user/campaigns/updating_campaigns"
+                                            rel="noopener noreferrer"
+                                            target="_blank"
+                                        >
+                                            docs on updating campaigns
```
Comment (mrnugget) [inline web/src/enterprise/campaigns/detail/CampaignDetails.tsx:466]:
```suggestion
                                            documentation on updating campaigns
```

File: web/src/enterprise/campaigns/detail/CampaignDetails.tsx [hunk 4]
```diff
@@ -425,6 +425,51 @@ export const CampaignDetails: React.FunctionComponent<Props> = ({
...
+                                        </a>
+                                        .
```
Comment (mrnugget) [inline web/src/enterprise/campaigns/detail/CampaignDetails.tsx:468]:
```suggestion
                                        {' '}for more information.
```

**Comment by eseliger** [comment]
🍒 @uwedeportivo cherry-picked this as 4bc24c329e5c4e2eff974aa3497b9df3c024c7cd onto 3.15

**Pull request merged**